src/openforms/formio/formatters/tests/files/all_components_data.json

```diff
@@ -17,11 +17,22 @@
   },
   "mapLineString": {
     "type": "LineString",
-    "coordinates": [[4.893, 52.366], [4.894, 52.367], [4.895, 52.368]]
+    "coordinates": [
+      [4.893, 52.366],
+      [4.894, 52.367],
+      [4.895, 52.368]
+    ]
```

Comment:
Keep these formatting changes out of the PR please, they're completely unrelated.